common/workunit/workunit.cpp

```diff
@@ -2917,6 +3066,11 @@ void WuScopeFilter::finishedFilter()
             if (!(properties & (PTattributes|PThints)))
                 sourceFlags &= ~(SSFsearchGraph|SSFsearchWorkflow);
         }
+
+        if (!(properties & PTnotes))
+            sourceFlags &= ~SSFsearchExceptions;
+        else
```

Comment:
I don't think the else is needed.  When I tested with properties[note] (after adding it to the table) the value was output by wutool.

Reply:
Is the problem that w1, w1:graph1 and w1:graph1:sg1 are not output?

Reply:
Nope, the problem is that if only the 'notes' is selected, then all the other iterators are added as well. Also, "global" and "workflow" scopes (that have no notes) are being returned when only "notes" are requested

Reply:
How are you requesting the information?

Reply:
I had PropertiesToReturn=PTnotes,  maxDepth=0 and nestedDepth=9999.  My mistake I should have set maxDepth=999. My mistake.  (SSFsearchGlobalStats not required when searching PTnotes specified)